sympy/stats/tests/test_joint_rv.py

```diff
@@ -354,3 +352,23 @@ def test_sample_seed():
                 assert all(s1 != s2)
         except NotImplementedError:
             continue
+
+
+def test_issue_21057():
+    m = Normal("x", [0, 0], [[0, 0], [0, 0]])
+    n = MultivariateNormal("x", [0, 0], [[0, 0], [0, 0]])
+    p = Normal("x", [0, 0], [[0, 0], [0, 1]])
+    assert m == n
+    libraries = ['scipy', 'numpy', 'pymc3']
+    for library in libraries:
+        try:
+            imported_lib = import_module(library)
+            if imported_lib:
+                s1 = sample(m, size=8)
+                s2 = sample(n, size=8)
+                s3 = sample(p, size=8)
```

Comment:
This doesn't pass the library argument to sample so only scipy is tested. The test actually fails for pymc3. I've fixed the test in #23512 but I had to XFAIL the pymc3 part because it doesn't work.